cloudbuild.yaml

```diff
@@ -0,0 +1,10 @@
+steps:
+- name: golang:1.21
```

Comment:
Do we want to use a fix tag ? Not sure about backward compatibility knowing `ko` will use a superior version
https://github.com/ko-build/ko/blob/0dcace336cc5f723150d77487dbaf56f849cc361/go.mod#L3

Reply:
for 1.21+ it will respect toolchain

Reply:
I think we can iterate on this, though eventually I'd prefer we be pinning images with a digest to avoid risk of dockerhub / golang dockerhub compromise.